lib/SmartyPlugins/block.box_header.php

```diff
@@ -40,7 +40,7 @@ function smarty_block_box_header($params, $content, $template, $repeat)
             return '
 				<div' . ($id ? ' id="' . $id . '"' : '') . ' class="lms-ui-box-header">
 					' . (strpos($icon, '/') !== false ? '<IMG src="' . $icon . '" alt="">'
-                        : '<i class="' . $icon . '"></i>')
+                        : '<i class="lms-ui-icon-' . $icon . '"></i>')
```

Comment:
Lepiej zrobić, żeby działało zarówno **location** jak i **lms-ui-icon-location**. To już jest w innych funkcjach Smarty - powinno być łatwe do przeniesienia tutaj.

Reply:
Zrobione.